packages/mobile/src/send/InviteRewardsBanner.tsx

```diff
@@ -1,22 +1,46 @@
 import colors from '@celo/react-components/styles/colors'
 import fontStyles from '@celo/react-components/styles/fonts'
 import * as React from 'react'
-import { useTranslation } from 'react-i18next'
-import { StyleSheet, Text, View } from 'react-native'
+import { Trans, useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+import { INVITE_REWARDS_TERMS_LINK } from 'src/config'
 import { Namespaces } from 'src/i18n'
-import Invite from 'src/icons/Invite'
+import { notificationInvite } from 'src/images/Images'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import useSelector from 'src/redux/useSelector'
 
 export function InviteRewardsBanner() {
   const { t } = useTranslation(Namespaces.inviteFlow11)
+  const rewardAmount = useSelector((state) => state.send.inviteRewardCusd)
+
+  const openInviteTerms = () => {
+    navigate(Screens.WebViewScreen, { uri: INVITE_REWARDS_TERMS_LINK })
+  }
 
   return (
-    <View style={styles.container} testID="InviteRewardsBanner">
-      <Invite />
+    <TouchableOpacity
```

Comment:
We should use `Touchable` instead so it uses the appropriate platform effect on iOS and Android,